tests/integration_tests/test_account.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from eth_account import Account
+
+from .utils import ADDRS, w3_wait_for_new_blocks
+
+
+def test_get_transaction_count(ethermint):
```

Comment:
Could you also include the websockets in the test as well? You can look at the `test_types.py` file under the same directory to see how, think that the input can simply be changed to `ethermint_rpc_ws` instead of just `ethermint`.

Reply:
will miss crosscheck with [geth](https://github.com/evmos/ethermint/blob/5f009aeece196eda008f34a0d0aa96ccfe0fd990/tests/integration_tests/test_account.py#L9) if replace with [ethermint_rpc_ws](https://github.com/evmos/ethermint/blob/5f009aeece196eda008f34a0d0aa96ccfe0fd990/tests/integration_tests/conftest.py#L53)